server/cmake/modules/FindLibClang.cmake

```diff
@@ -61,20 +61,20 @@ endforeach()
 
 find_path(LIBCLANG_INCLUDE_DIR clang-c/Index.h
   PATHS ${libclang_llvm_header_search_paths}
-  PATH_SUFFIXES LLVM/include #Windows package from http://llvm.org/releases/
+  PATH_SUFFIXES llvm/include #Windows package from http://llvm.org/releases/
```

Comment:
irony-server was unable to build without this change or you just saw that it was lowercase on your system and changed?

We may need both variant eventually.